TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -315,7 +331,12 @@ extension HomeOptionsCardViewController: UISearchBarDelegate {
     }
 
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
-        searchBar.showsCancelButton = true
+        if searchText.isEmpty {
+            searchBar.returnKeyType = .default
+        } else {
+            searchBar.returnKeyType = .search
+        }
```

Comment:
can shorten to `searchBar.returnKeyType = searchText.isEmpty ? .default : .search`